src/components/Explore/TokenTable/TokenRow.tsx

```diff
@@ -263,10 +265,19 @@ const VolumeCell = styled(DataCell)`
 `
 /* Loading state bubbles */
 const LoadingBubble = styled.div`
-  background-color: ${({ theme }) => theme.backgroundContainer};
   border-radius: 12px;
   height: 24px;
   width: 50%;
+  animation: ${loadingAnimation} 1.5s infinite;
+  animation-fill-mode: both;
```

Comment:
did you copy paste this stuff? i would add it to the file you got the loading animation from